### PR TITLE
Test against (upcoming) PHP 7.3 on Travis

### PR DESCRIPTION
**Pull request by WyriHaximus**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,7 +29,7 @@ before_script:
   - if [[ $STATIC_ANALYSIS = true ]]; then composer require phpstan/phpstan --no-update; fi
   - composer install
   - set -e # Stop on first error.
-  - phpenv config-rm xdebug.ini
+  - phpenv config-rm xdebug.ini || true
```
Comment (roblourens) [inline .travis.yml]: Why this change?

Comment (TysonAndre) [reply .travis.yml]: Xdebug doesn't support PHP 7.3 yet, so it isn't preinstalled, so there's nothing to remove, so that returns false and would otherwise fail the build.

Comment (WyriHaximus) [reply .travis.yml]: That example, it's the reason the first build fails. Here to be precise: https://travis-ci.org/Microsoft/tolerant-php-parser/jobs/448491725#L540

Comment (roblourens) [reply .travis.yml]: Oh, got it. Thanks!
